corehq/sql_db/connections.py

```diff
@@ -1,8 +1,10 @@
+from __future__ import division
 from __future__ import absolute_import
 from __future__ import unicode_literals
-import random
+from numpy import random
```

Comment:
why the switch to numpy?

Reply:
Currently the strategy being used to do weighted random balancing is to explode the list with an item the number of times promotional to the weight. numpy has a builtin method for this. This avoids going for that step of exploding the list and also allows us to remove restriction on weight to be an integer